### PR TITLE
Add exception handling for invalid JWT tokens

### PR DESCRIPTION
**Pull request by dolfelt**
When passing a non-JWT string it causes an unhandled exception.

cr @shadowhand @elazar

### CODE REVIEW

File: tests/Jwt/FirebaseParserTest.php [hunk 1]
```diff
@@ -65,6 +65,20 @@ public function testParseTokenWithExpiredToken()
...
+    public function testParseTokenWithInvalidToken()
+    {
+        $parser = new FirebaseParser($this->config);
+
+        try {
+            $parser->parseToken(
+                'undefined'
+            );
+            $this->fail('Expected exception was not thrown');
+        } catch (InvalidException $e) {
```
Comment (shadowhand) [inline tests/Jwt/FirebaseParserTest.php]: Rather than using a try/catch, use annotations?

Comment (dolfelt) [inline tests/Jwt/FirebaseParserTest.php]: This is how all the other tests were done. I'd rather not mix and match styles.

Comment (shadowhand) [inline tests/Jwt/FirebaseParserTest.php]: Keep a bad precedent just to be consistent?

Comment (elazar) [inline tests/Jwt/FirebaseParserTest.php]: @shadowhand It's a stylistic difference, and I don't think it's one that warrants blocking this PR. If the difference bothers you that much, you can always submit a separate PR to handle those changes. :wink:

**Comment by shadowhand** [comment]
:+1:
